Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -27,6 +38,83 @@ void RegExpConstructor::initialize(Realm& realm)
     define_native_accessor(realm, *vm.well_known_symbol_species(), symbol_species_getter, {}, Attribute::Configurable);
 
     define_direct_property(vm.names.length, Value(2), Attribute::Configurable);
+
+    // Additional properties of the RegExp constructor, https://github.com/tc39/proposal-regexp-legacy-features#additional-properties-of-the-regexp-constructor
+    define_native_accessor(realm, "input", input_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$_", input_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "lastMatch", last_match_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$&", last_match_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "lastParen", last_paren_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$+", last_paren_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "leftContext", left_context_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$`", left_context_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "rightContext", right_context_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$'", right_context_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$1", group_1_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$2", group_2_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$3", group_3_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$4", group_4_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$5", group_5_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$6", group_6_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$7", group_7_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$8", group_8_getter, {}, Attribute::Configurable);
+    define_native_accessor(realm, "$9", group_9_getter, {}, Attribute::Configurable);
+}
+
+// UpdateLegacyRegExpStaticProperties ( C, S, startIndex, endIndex, capturedValues )
+// https://github.com/tc39/proposal-regexp-legacy-features#updatelegacyregexpstaticproperties--c-s-startindex-endindex-capturedvalues-
```

Comment:
We always put the function signature and URL on the same line

```suggestion
// UpdateLegacyRegExpStaticProperties ( C, S, startIndex, endIndex, capturedValues ), https://github.com/tc39/proposal-regexp-legacy-features#updatelegacyregexpstaticproperties--c-s-startindex-endindex-capturedvalues-
```

Reply:
ok